verifier/Cargo.toml

```diff
@@ -7,17 +7,23 @@ description.workspace = true
 license.workspace = true
 
 [dependencies]
-anyhow.workspace = true
-plonky2 = { workspace = true } #, features = ["no_random"] }
-wormhole-circuit = { path = "../circuit" }
+anyhow = { workspace = true }
+plonky2 = { workspace = true, default-features = false }
+hex = { workspace = true }
+rand = { workspace = true }
+wormhole-circuit = { path = "../circuit", default-features = false }
 
 [dev-dependencies]
-wormhole-prover = { path = "../prover" }
 criterion = { workspace = true }
-plonky2 = { workspace = true }
+wormhole-prover = { path = "../prover" }
 
 [features]
-
-[[bench]]
-name = "verifier_bench"
-harness = false
```

Comment:
Did we mean to remove the bench?

Reply:
Moved it to the tests crate as well, it seems to work now